crates/cargo-test-support/Cargo.toml

```diff
@@ -5,6 +5,8 @@ authors = ["Alex Crichton <alex@alexcrichton.com>"]
 license = "MIT OR Apache-2.0"
 edition = "2018"
 
+build = "build.rs"
```

Comment:
It's ok to omit this (since it's the default)

Reply:
True, thank you for the pointer :)